lib/spack/spack/util/prefix.py

```diff
@@ -60,17 +60,18 @@ class Prefix(str):
 
     def __new__(cls, path):
         s = super(Prefix, cls).__new__(cls, path)
-        s.bin     = join_path(s, 'bin')
-        s.bin64   = join_path(s, 'bin64')
-        s.sbin    = join_path(s, 'sbin')
-        s.etc     = join_path(s, 'etc')
-        s.include = join_path(s, 'include')
-        s.lib     = join_path(s, 'lib')
-        s.lib64   = join_path(s, 'lib64')
-        s.libexec = join_path(s, 'libexec')
-        s.share   = join_path(s, 'share')
-        s.doc     = join_path(s.share, 'doc')
-        s.info    = join_path(s.share, 'info')
+        s.bin       = join_path(s, 'bin')
+        s.bin64     = join_path(s, 'bin64')
+        s.sbin      = join_path(s, 'sbin')
+        s.etc       = join_path(s, 'etc')
+        s.include   = join_path(s, 'include')
+        s.include64 = join_path(s, 'include64')
```

Comment:
OT: I wonder why in this file we have +12/-11 lines diff, while at a first glance it seems you aded only one line...